common/hooks/post-install/14-fix-permissions.sh

```diff
@@ -0,0 +1,28 @@
+# This hook fixes permissions in common places
+
+change_file_perms() {
+	dir="$PKGDESTDIR$1"
```

Comment:
```suggestion
	dir="${PKGDESTDIR}${1}"
```

To make reading easier, since it's appending.